dask_ml/_partial.py

```diff
@@ -167,8 +168,17 @@ def fit(model, x, y, compute=True, shuffle_blocks=True, random_state=None, **kwa
     if y is not None:
         if not hasattr(y, "chunks") and hasattr(y, "to_dask_array"):
             y = y.to_dask_array()
+
+    # dask 1.0.0 has a dd.DataFrame.to_dask_array, but not earlier versions
```

Comment:
I think we're calling 1.0.0 our "minimum dask version" for a while. So feel free to use it.

So IIUC, this shouldn't be necessary, since the `if not hasattr(y, "chunks") and hasattr(y, "to_dask_array")` will handle things.

Reply:
Thanks, that makes life a little easier.